app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardScheduleStep/PlanWizardScheduleStep.js

```diff
@@ -13,14 +18,20 @@ export const PlanWizardScheduleStep = ({ targetProvider, migration_plan_type_rad
       label={__('Migration Type')}
       inline_label={false}
       controlWidth={12}
+      info={
```

Comment:
This info icon should instead be a warning icon (https://github.com/ManageIQ/manageiq-v2v/blob/master/app/javascript/react/screens/App/common/forms/FormField.js#L92 should render `name="warning-triangle-o"` on that `Icon`), so we probably need to add a prop like `infoIconName` to `FormField` that defaults to `'info'` but can be overridden here.

Reply:
Yes sir, fixed & updated the screenshots as well.